File: Lancetilla BackEnd/Lancetilla/Lancetilla.API/Models/PlantasViewModel.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Threading.Tasks;

namespace Lancetilla.API.Models
{
    public class PlantasViewModel
    {
        public int plan_Id { get; set; }
        public string plan_Codigo { get; set; }
        public int tipl_Id { get; set; }
        public string tipl_NombreComun { get; set; }
        public string tipl_NombreCientifico { get; set; }
        public int arbo_Id { get; set; }
        public string arbo_Descripcion { get; set; }
        public string usua_UserCreaNombre { get; set; }
        public int? plan_UserCreacion { get; set; }
        public DateTime? plan_FechaCreacion { get; set; }
        public string usua_UserModiNombre { get; set; }
        public int? plan_UserModificacion { get; set; }
        public DateTime? plan_FechaModificacion { get; set; }
        public bool? plan_Estado { get; set; }

    }
}

File: Lancetilla FrontEnd/src/app/demo/Models/PlantasViewModel.ts
export class PlantasViewModel {
    constructor(
        public plan_Id?: number,
        public plan_Codigo?: string,
        public arbo_Id?: number,
        public tipl_Id?: number,
        public arbo_Descripcion?: string,
        public tipl_Descripcion?: string,
        public usua_UserCreaNombre?: string,
        public plan_UserCreacion?: number,
        public plan_FechaCreacion?: string,
        public usua_UserModiNombre?: string,
        public plan_UserModificacion?: number,
        public plan_FechaModificacion?: string,
        public plan_Estado?: boolean
    ) { }
}
